spec/abstractops.html

```diff
@@ -196,31 +196,48 @@ <h1>ToShowOffsetOption ( _normalizedOptions_ )</h1>
     <h1>
       ToTemporalRoundingIncrement (
         _normalizedOptions_: an Object,
-        _dividend_: an integer or *undefined*,
+      ): either a normal completion containing a Number, or an abrupt completion
+    </h1>
+    <dl class="header">
+      <dt>description</dt>
+      <dd>
+        It extracts the value of the property named *"roundingIncrement"* from _normalizedOptions_, makes sure it is a finite Number of at least 1, and returns that value.
+        It performs no further validation.
+      </dd>
+    </dl>
+    <emu-alg>
+      1. Let _increment_ be ? GetOption(_normalizedOptions_, *"roundingIncrement"*, *"number"*, *undefined*, *1*<sub>𝔽</sub>).
+      1. If _increment_ is *NaN* or *+&infin;*<sub>𝔽</sub>, throw a *RangeError* exception.
+      1. If _increment_ &lt; *1*<sub>𝔽</sub>, throw a *RangeError* exception.
+      1. Return _increment_.
+    </emu-alg>
+  </emu-clause>
+
+  <emu-clause id="sec-validatetemporalroundingincrement" type="abstract operation">
+    <h1>
+      ValidateTemporalRoundingIncrement (
+        _increment_: a Number,
+        _dividend_: an integer &geq; 1,
```

Comment:
```suggestion
        _dividend_: a positive integer,
```
(cf. [CodePointAt](https://tc39.es/ecma262/multipage/ecmascript-language-source-code.html#sec-codepointat))

Also, it seems strange for an operation to accept a Number and an integer... should it be refactored to avoid that?

Reply:
I agree, it should be refactored to avoid that. I explained a bit about this in the commit message, but tl;dr is that I'd like ToTemporalRoundingIncrement to convert to ℝ and truncate, but I don't think that can be achieved without a normative change (which I'll do as part of #2254).